### PR TITLE
dplyr 1 0

### PR DESCRIPTION
**Pull request by gergness**
Fixes for the upcoming version of dplyr, and removes warnings in tests about deprecated dplyr functions.

`filter` has been updated to better match modern dplyr, but `summarize` is still a kind of unique beast, it works for basic cases, but the non-standard evaluation isn't very robust (you can't for example use `my_mean <- function(x) mean(x, na.rm)` and use `my_mean()` in a summarize call.

### CODE REVIEW

File: DESCRIPTION [hunk 1]
```diff
@@ -7,7 +7,7 @@ Description: In order to facilitate analysis of datasets hosted on the Crunch
     'group_by', 'summarize', and 'collect' are implemented in such a way as to
     perform as much computation on the server and pull as little data locally
     as possible.
-Version: 0.3.2
+Version: 0.3.2.9000
```
Comment (malecki) [inline DESCRIPTION]: master is already at 0.3.3 apparently?

File: DESCRIPTION [hunk 2]
```diff
@@ -7,7 +7,7 @@ Description: In order to facilitate analysis of datasets hosted on the Crunch
...
 Authors@R: c(
     person("Greg", "Freedman Ellis", role="aut", email="greg@crunch.io"),
```
Comment (malecki) [inline DESCRIPTION]:
```suggestion
    person("Greg", "Freedman Ellis", role=c("aut","cre"), email="greg@crunch.io"),
```

Comment (malecki) [reply DESCRIPTION]: (and take it off of mine)

**Review by malecki — CHANGES_REQUESTED** [review]

**Review by gergness — COMMENTED** [review]
rename `as_crubble` -> `as_cr_tibble`

**Pull request merged**